doc/api/fs.md

```diff
@@ -1668,6 +1668,10 @@ changes:
   - version: v2.3.0
     pr-url: https://github.com/nodejs/node/pull/1845
     description: The passed `options` object can be a string now.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: The `fs` options allow overriding the used `fs`
+                 implementation.
```

Comment:
Late nit: it would be ideal to keep the versioning order the next time.

Reply:
versioning order?

Reply:
The changes entry should be at the top but it's at the bottom of the changes.

Reply:
Ah, my mistake. I though it was the other way around.

Reply:
There is no clear rule for it so far. So it's different each time and it's just good to keep it consistent in the changes itself. We might want to define an order at some point.